internal/integration/cli/mounts.go

```diff
@@ -25,7 +25,7 @@ func (suite *MountsSuite) SuiteName() string {
 // TestSuccess verifies successful execution.
 func (suite *MountsSuite) TestSuccess() {
 	suite.RunCLI([]string{"mounts", "--nodes", suite.RandomDiscoveredNode()},
-		base.StdoutShouldMatch(regexp.MustCompile(`FILESYSTEM`)))
+		base.StdoutShouldMatch(regexp.MustCompile(`FILESYSTEM.*/var/lib/extra`)))
```

Comment:
unfortunately we can't do this, as this won't work for any cluster provisioned outside of `e2e-qemu`.

I think we need leave this test as is, but rather another test which is only enabled on some flag/environment variable, and set this flag when running e2e-qemu test, and we can verify that mountpoints are present with expected names